<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PanelConnectomeMatrix</class>
 <widget class="QScrollArea" name="PanelConnectomeMatrix">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>361</width>
    <height>860</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ScrollArea</string>
  </property>
  <property name="frameShape">
   <enum>QFrame::NoFrame</enum>
  </property>
  <property name="widgetResizable">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="scrollAreaWidgetContents">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>361</width>
     <height>860</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>6</number>
    </property>
    <property name="leftMargin">
     <number>5</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>5</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <layout class="QGridLayout" name="gridLayout" columnstretch="0,1">
      <property name="verticalSpacing">
       <number>6</number>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="labelFileName">
        <property name="text">
         <string>File name</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="lineEditFileName">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <spacer name="verticalSpacer_2">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Fixed</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>5</width>
        <height>0</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="Line" name="line_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>From:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTreeWidget" name="treeWidgetFrom">
      <property name="showDropIndicator" stdset="0">
       <bool>false</bool>
      </property>
      <property name="selectionMode">
       <enum>QAbstractItemView::ExtendedSelection</enum>
      </property>
      <property name="rootIsDecorated">
       <bool>false</bool>
      </property>
      <property name="itemsExpandable">
       <bool>false</bool>
      </property>
      <attribute name="headerVisible">
       <bool>false</bool>
      </attribute>
      <column>
       <property name="text">
        <string notr="true">1</string>
       </property>
      </column>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Opacity</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSlider" name="sliderFromOpacity">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>1</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximum">
         <number>100</number>
        </property>
        <property name="value">
         <number>0</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDoubleSpinBox" name="doubleSpinBoxFromOpacity">
        <property name="maximum">
         <double>1.000000000000000</double>
        </property>
        <property name="singleStep">
         <double>0.100000000000000</double>
        </property>
        <property name="value">
         <double>0.000000000000000</double>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="Line" name="line">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="verticalSpacer_3">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Fixed</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>1</width>
        <height>5</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>To:</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>5</width>
          <height>5</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QCheckBox" name="checkBox">
        <property name="text">
         <string>All</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton">
        <property name="maximumSize">
         <size>
          <width>35</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>9</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Clear</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QTreeWidget" name="treeWidgetTo">
      <property name="showDropIndicator" stdset="0">
       <bool>false</bool>
      </property>
      <property name="selectionMode">
       <enum>QAbstractItemView::ExtendedSelection</enum>
      </property>
      <property name="rootIsDecorated">
       <bool>false</bool>
      </property>
      <property name="itemsExpandable">
       <bool>false</bool>
      </property>
      <attribute name="headerVisible">
       <bool>false</bool>
      </attribute>
      <column>
       <property name="text">
        <string notr="true">1</string>
       </property>
      </column>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Opacity</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSlider" name="sliderToOpacity">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>1</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximum">
         <number>100</number>
        </property>
        <property name="value">
         <number>100</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDoubleSpinBox" name="doubleSpinBoxToOpacity">
        <property name="maximum">
         <double>1.000000000000000</double>
        </property>
        <property name="singleStep">
         <double>0.100000000000000</double>
        </property>
        <property name="value">
         <double>1.000000000000000</double>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="Line" name="line_3">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QGridLayout" name="gridLayout_2" columnstretch="0,1">
      <property name="verticalSpacing">
       <number>5</number>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Spline radius</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QLineEdit" name="lineEditSplineRadius">
          <property name="maximumSize">
           <size>
            <width>60</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>0.5</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>Spline color</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QtColorPicker" name="colorPickerSpline" native="true"/>
        </item>
        <item>
         <spacer name="horizontalSpacer_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>10</width>
        <height>10</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QtColorPicker</class>
   <extends>QWidget</extends>
   <header>qtcolorpicker.h</header>
   <container>1</container>
   <slots>
    <signal>colorChanged(QColor)</signal>
    <signal>currentColorChanged(QColor)</signal>
   </slots>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>treeWidgetFrom</sender>
   <signal>itemSelectionChanged()</signal>
   <receiver>PanelConnectomeMatrix</receiver>
   <slot>OnCurrentFromChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>229</x>
     <y>298</y>
    </hint>
    <hint type="destinationlabel">
     <x>463</x>
     <y>421</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>PanelConnectomeMatrix</receiver>
   <slot>OnCheckBoxToAll(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>289</x>
     <y>482</y>
    </hint>
    <hint type="destinationlabel">
     <x>415</x>
     <y>437</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>treeWidgetTo</sender>
   <signal>itemSelectionChanged()</signal>
   <receiver>PanelConnectomeMatrix</receiver>
   <slot>OnCurrentToChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>223</x>
     <y>534</y>
    </hint>
    <hint type="destinationlabel">
     <x>528</x>
     <y>550</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sliderFromOpacity</sender>
   <signal>valueChanged(int)</signal>
   <receiver>PanelConnectomeMatrix</receiver>
   <slot>OnSliderFromOpacity(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>165</x>
     <y>432</y>
    </hint>
    <hint type="destinationlabel">
     <x>471</x>
     <y>449</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sliderToOpacity</sender>
   <signal>valueChanged(int)</signal>
   <receiver>PanelConnectomeMatrix</receiver>
   <slot>OnSliderToOpacity(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>172</x>
     <y>713</y>
    </hint>
    <hint type="destinationlabel">
     <x>596</x>
     <y>772</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>pushButton</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>302</x>
     <y>482</y>
    </hint>
    <hint type="destinationlabel">
     <x>331</x>
     <y>483</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>treeWidgetTo</receiver>
   <slot>clearSelection()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>331</x>
     <y>492</y>
    </hint>
    <hint type="destinationlabel">
     <x>335</x>
     <y>541</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lineEditSplineRadius</sender>
   <signal>textChanged(QString)</signal>
   <receiver>PanelConnectomeMatrix</receiver>
   <slot>OnLineEditSplineRadius(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>125</x>
     <y>767</y>
    </hint>
    <hint type="destinationlabel">
     <x>129</x>
     <y>901</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>OnCurrentFromChanged()</slot>
  <slot>OnCurrentToChanged()</slot>
  <slot>OnCheckBoxToAll(bool)</slot>
  <slot>OnSliderFromOpacity(int)</slot>
  <slot>OnSliderToOpacity(int)</slot>
  <slot>OnLineEditSplineRadius(QString)</slot>
 </slots>
</ui>
